support/sql/general/support-policy-hardware-virtualization-product.md

```diff
@@ -172,13 +172,21 @@ The following restrictions and limitations may affect the support policy of the
 
     - SQL Server FCI n Windows Server 2016 and later versions with Azure shared disks. For more information review [Create an FCI with Azure shared disks (SQL Server on Azure VMs)](/azure/azure-sql/virtual-machines/windows/failover-cluster-instance-azure-shared-disks-manually-configure).
 
-- **Q9: Can customers run SQL Server in the Microsoft Azure VM role?**
+- **Q9: Does Microsoft support SQL Server running in Microsoft Azure VMware Solution?**
 
-  A9:  Microsoft Azure VM role is a non-persistent role and not the same as Microsoft Azure Virtual Machine. It is not supported for SQL Server production use. Customers who want to deploy data platform capabilities today in the Microsoft Azure platform should use Microsoft Azure Virtual Machine or Microsoft Azure SQL Database.
+  A9: Yes. Microsoft supports SQL Server 2008 and later versions in Microsoft Azure Infrastructure Services that includes Microsoft Azure VMware Solution. When you deploy SQL Server 2008 and later versions in Microsoft Azure VMware Solution, see:
```

Comment:
Change to: "Yes, Microsoft supports..."

Also, replace: 

- "Services that includes..." with "Services, which includes".
- "....later versions in Microsoft Azure VMware Solution, see:" with "...later versions in Microsoft Azure VMware Solution, consider the following resources:"